app/views/travel_advice/index.html.erb

```diff
@@ -33,10 +33,10 @@
                 Check the <a href="/guidance/national-lockdown-stay-at-home">rules that apply to you in England</a>, <a href="https://www.gov.scot/coronavirus-covid-19/">Scotland</a>, <a href="https://gov.wales/covid-19-alert-levels">Wales</a> and <a href="https://www.nidirect.gov.uk/articles/coronavirus-covid-19-regulations-guidance-what-restrictions-mean-you">Northern Ireland</a>.
               </p>
               <p class="govuk-body">
-                If you intend to <a href="/guidance/coronavirus-covid-19-testing-for-people-travelling-to-england">travel to England</a>, <a href="https://www.gov.scot/publications/coronavirus-covid-19-public-health-checks-at-borders/pages/testing-for-people-travelling-to-scotland/">Scotland</a>, or <a href="https://gov.wales/new-mandatory-coronavirus-testing-for-travellers-into-wales">Wales</a> from abroad, including UK nationals returning home from travel abroad, you must provide evidence of a negative COVID-19 test result taken up to 3 days before departure. If you do not comply (and you do not have a valid exemption) your airline or carrier may refuse you boarding and/or you may be fined on arrival.
+                If you intend to <a href="/guidance/coronavirus-covid-19-testing-for-people-travelling-to-england">travel to England</a>, <a href="https://www.gov.scot/publications/coronavirus-covid-19-public-health-checks-at-borders/pages/testing-for-people-travelling-to-scotland/">Scotland</a>, or <a href="https://gov.wales/coronavirus-covid-19-testing-people-travelling-wales">Wales</a> from abroad, including UK nationals returning home, you must provide evidence of a negative COVID-19 test result taken up to 3 days before departure. If you do not comply (and you do not have a valid exemption) your airline or carrier may refuse you boarding and/or you may be fined on arrival.
               </p>
               <p class="govuk-body">
-                Before you return to the UK you must <a href="/provide-journey-contact-details-before-travel-uk">provide your journey and contact details</a>. You must <a href="/uk-border-control">self-isolate</a> when you enter the UK from any foreign country except Ireland, unless you have a valid exemption.
+                Before you return to the UK you must <a href="/provide-journey-contact-details-before-travel-uk">provide your journey and contact details</a>. You must <a href="/uk-border-control">self-isolate</a> when you enter UK from any foreign country except Ireland, unless you have a valid exemption.
```

Comment:
`when you enter UK`... I know this was like this on the copy but I am not sure if this is an error or not. Will check!

Reply:
Thanks! Though it's the same on https://github.com/alphagov/government-frontend/pull/1988 😬

Reply:
Re-add the "the". Also added a PR to updated government-frontend: https://github.com/alphagov/government-frontend/pull/1989